app/forms/TraderEoriNumberFormProvider.scala

```diff
@@ -50,7 +50,5 @@ class TraderEoriNumberFormProvider @Inject() extends Mappings {
             Invalid(ValidationError("provideTraderEori.error.specialCharacters"))
           case _                                 => Invalid(ValidationError("provideTraderEori.error.default"))
         }
-      case _ =>
-        Invalid(ValidationError("provideTraderEori.error.default"))
```

Comment:
Unreachable

Reply:
Is it unreachable because Constraint is always a string?